data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -341,6 +375,20 @@ def _prepare_initial_df(self) -> pd.DataFrame:
                 df_copy[col] - min_value
             ) / (max_value - min_value)
 
+        # Special logic: create a combined population field.
```

Comment:
These seems like something I'd love to have a test for.

Reply:
Yeah definitely agreed. 😢  Some day soon!!